### PR TITLE
Update 4.8 to allow for more flexible nomination and approval of fellows

### PR DESCRIPTION
**Pull request by VanL**

### CODE REVIEW

**Comment by jefftriplett** [comment]
> Re: **Section 5.6. Resignation and Removal of Directors.** 

> Should we adopt a similar section for Fellows since we have had at least one Fellow step down over the last few years? Also, with the CoC in mind, it might be appropriate to remove a Fellow. 

This is resolved via: **"4.1 clarifies a Fellow is a class of Member"**

**Comment by lorenanicole** [comment]
> Re: **Section 5.6. Resignation and Removal of Directors.** 
> 
> 
> 
> Should we adopt a similar section for Fellows since we have had at least one Fellow step down over the last few years? Also, with the CoC in mind, it might be appropriate to remove a Fellow. 

Oh I am 👍🏽 this! @VanL would that language look similar in spirit to the Director stepping down language?

File: bylaws.md [hunk 1]
```diff
@@ -330,18 +330,18 @@ The Board of Directors may, from time to time, create alternate
...
-or vampirism. Fellows may exercise voting rights at any time, and if
-they vote, shall be counted for purposes of a quorum.
+a member must be nominated by any Member of the corporation other
+than a Basic Member, or by a specially chartered Working Group, which 
+nomination should be based upon certain criteria to be established by 
+the Board of Directors and which criteria shall be designed to emphasize 
+extraordinary contributions. Following such nomination, approval of two-
+thirds (2/3) of the members entitled to vote, or two-thirds (2/3) of the 
+members of the chartered Working Group, or two-thirds (2/3) of the Board of 
+Directors,shall be required shall be required in order for a member to become
```
Comment (di) [inline bylaws.md]:
```suggestion
Directors, shall be required shall be required in order for a member to become 
```

Comment (jefftriplett) [reply bylaws.md]: We have an extra "hall be required" phrase in here.

```suggestion
Directors, shall be required in order for a member to become 
```

Comment (ewdurbin) [inline bylaws.md]:
```suggestion
Directors, shall be required in order for a member to become 
```

**Pull request merged**